packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -249,34 +249,34 @@ exports[`Podcast Promo Examples basic example 1`] = `
   }
 }
 
-.emotion-26 {
+.emotion-24 {
   padding: 0.5rem;
   display: inline-block;
   max-width: calc(100% - 96px);
```

Comment:
What's this going output? % or px?

Reply:
With `calc()` the value parsed by the browser is the actual `calc()` expression. i.e.

```css
// Value specified in CSS
.foo {
    width: calc(100% - 50px);
}

// Computed value in browser
.foo {
    width: calc(100% - 50px);
}
```

I actually do want to dig into this `max-width` value... I need to ask @ryanmccombe why we need it.

Reply:
Separate to your query Catharine but an improvement was to use flex instead of max-width, so these calcs are gone 💥